conda-forge.yml

```diff
@@ -1,4 +1,8 @@
+azure:
+  free_disk_space: true
```

Comment:
As one the [CI builds]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=763625&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=c7d0fcb4-3a10-5b26-dfea-e879a93fc2ab&l=1592 ) (also [attached log]( https://github.com/conda-forge/faiss-split-feedstock/files/12384310/faiss-split-feedstock_20230816.1_linux_aarch64_cuda_compiler_version11.2cxx_compiler_version10.txt )) ran out of space, try cleaning up the images first